src/identity/hcs/did/event/hcs-did-event-target-name.ts

```diff
@@ -1,4 +1,5 @@
 export enum HcsDidEventTargetName {
+    DID_DOCUMENT = "DIDDocument",
```

Comment:
@zbarbuto my knowledge is a little outdated but is `Document` target used anywhere at all?

Reply:
From what I can tell, it is used for the `DELETE` operation only (at least, that's the only place where it is referenced).

Reply:
Maybe `DID_DOCUMENT` could be used for both then and `Document` key could simply be removed? 
Only if that does not impose too many changes around the rest of the codebase.